webpages/settings/components/category-selector.html

```diff
@@ -28,6 +28,8 @@
     cursor: pointer;
     display: flex;
     align-items: center;
+    margin: 10px;
```

Comment:
```suggestion
    margin: 10px;
    margin-bottom: 0;
```
This will fix the issue with small categories: https://github.com/ScratchAddons/ScratchAddons/pull/7279#issuecomment-2051613850